onnx/test/shape_inference_test.py

```diff
@@ -1053,6 +1049,41 @@ def test_onehot_with_axis(self):  # type: () -> None
             [])
         self._assert_inferred(graph, [make_tensor_value_info('Y', TensorProto.FLOAT, (2, None, 3, 5))])  # type: ignore
 
+    def test_loop(self):    # type: () -> None
+        # can't use self._make_graph for the subgraph as it add more inputs for the Reshape operations it inserts.
+        # this breaks the subgraph inferencing as it expects the number of inputs passed from Loop to match
+        # the GraphProto, but Loop knows nothing about the additional inputs.
+        input_value_infos = [make_tensor_value_info('iter_num_in', TensorProto.FLOAT, (1,)),
```

Comment:
Wonder if this is meant to be float or int?

Reply:
INT64 is more correct. Doesn't play any role in the inferencing though.